.github/workflows/codecov.yml

```diff
@@ -28,4 +28,4 @@ jobs:
           file: .build/cover.out
           exclude: ./
           token: ${{ secrets.CODECOV_TOKEN }}
-          slug: uber-go/cadence-client
```

Comment:
nit: There are some other references to the old repo in the documentation or comments. https://github.com/search?q=repo%3Acadence-workflow%2Fcadence-go-client%20uber-go%2Fcadence-client&type=code Do you mind fixing ti as well?

Reply:
yeah we should rename all those except the ones referring to package names. For now just unblocking codecov